packages/web-components/@sfx/autocomplete/package.json

```diff
@@ -18,5 +18,8 @@
     "directory": "packages/web-components/@sfx/autocomplete"
   },
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@sfx/events": "groupby/sfx-events"
```

Comment:
Should this point to the Github repo for now until we are able to publish `sfx-events` as a package? I'm guessing it will be waiting on a non-"sfx" name.